srcpkgs/xmonad-contrib/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'xmonad-contrib'
+pkgname=xmonad-contrib
+version=0.13
+revision=1
+build_style=haskell-stack
+stackage=lts-11.17
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXinerama-devel libXrandr-devel libXft-devel"
+short_desc="Third party extensions for xmonad"
+maintainer="xaltsc <xaltsc@protonmail.ch>"
+license="BSD-3-Clause"
+homepage="http://xmonad.org/"
```

Comment:
it is https://

Reply:
On the hackage homepage, the link is http://. Should I be canonical and follow this or change it to https anyway ?

Reply:
it just redirects http to https://, put https://

Reply:
Ok, done.